site/content/docs/user/rootless.md

```diff
@@ -52,6 +52,12 @@ Also, depending on the host configuration, the following steps might be needed:
   iptable_nat
   ```
 
+- If using podman, be aware that by default there is a [limit](https://docs.podman.io/en/v4.3/markdown/options/pids-limit.html#pids-limit-limit) to the number of pids that can be created. This can cause problems like nginx workers inside a container not spawning correctly.
```

Comment:
we should probably mention potential downsides as well? (to disabling the limit), users could also set a higher limit instead of no limit?

Reply:
What about something like...

If you want to raise this limit, edit your `containers.conf` file ( generally located in `/etc/containers/` ). Note that setting this to `0` will disable the limit, potentially allowing things like pid exhaustion to happen on the host machine. 
```ini
[containers]
pids_limit=10000
```

I'm not 100% sure about doing it this way because most people will likely copy the arbitrary number I chose for the block. 

Alternatively, I can just go with the current way, but rewrite it like this:

If you want to disable this limit, edit your `containers.conf` file ( generally located in `/etc/containers/` ). Note that setting could cause things like pid exhaustion to happen on the host machine. If you prefer, change `0` for your desired maximum number for the new limit. 

```ini
[containers]
pids_limit=0
```

Reply:
It seems reasonable to me to show an example using 0, along with that warning about pid exhaustion being a possibility.

But I could also see the inverse. Havintg10000 as the example at least does set some sort of limit. Then there could just be a note stating it is possible to have no limit with 0.

Either approach seems fine to me. Your latest update looks good.

Reply:
I agree.

I think we just need to at least mention the risk of exhaustion and the possibility to set a specific higher limit instead, with the alternate downside that it may not be high enough and/or may still enable exhaustion.

I think your sample with 0 is sufficient otherwise. Thank yu!